.buildkite/pipeline.arm64.yml

```diff
@@ -56,81 +56,32 @@
 #    - if [ "$BUILDKITE_BRANCH" == "master" ]; then python .buildkite/copy_files.py --destination wheels --path ./.whl; fi
 
 
-- label: ":mechanical_arm: :docker: Build Images: py37 [aarch64] (1/2)"
+- label: ":mechanical_arm: :docker: Build Images: {{matrix}} [aarch64] - cpu/cu115/cu116"
```

Comment:
Should we convert the versions to matrices as well?

Reply:
what do you mean ? you mean cpu/cuda versions?

I think they are structured this way because the wheel building part is shared. it is shard-ed into 2 build steps only for slightly faster build time.